README.md

```diff
@@ -1,26 +1,26 @@
 # XNU on Raspberry Pi 3 (BCM2837)
 
-I am trying to run XNU/Hackitosh, if possible. You can back me on this project :)
+I am trying to run XNU/Hackintosh, if possible. You can back me on this project :)
 
 ## Roadmap
 
 XNU Kernel:
 
 - [x] Compile Mach-O
 
-Then, there are two options. Either :
+Then, there are two options. Either:
 
 - [ ] Convert to an ELF Kernel
 - [ ] Load ELF Kernel
 
 or, we can write an Mach-O loader in ELF format, which can be loaded by Raspberry Pi:
 
-- [ ] Maybe a Mach-O loader
+- [ ] Maybe an Mach-O loader
```

Comment:
I think here we can keep the 'a' because it is the entire word Mach-O, pronounced as /mæk o/, but not M-A-C-H-O

Reply:
Apple uses "an":
https://developer.apple.com/documentation/appkit/nsworkspacelaunchconfigurationarchitecture

Reply:
Cool, I've read a book using "a"... Let's align with Apple